common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -16,6 +16,79 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     document
   }
 
+  private def documentWithVideos(videoUrls: Seq[String]): Document = {
+    val doc = "<html><body>" +
+      videoUrls.foldLeft(""){ (res: String, next: String) => res + s"""<figure class="element-video" data-canonical-url="$next"><iframe></iframe></figure>""" } +
```

Comment:
Rather than using `foldLeft` you can simple `map` your `Seq` of urls into a `Seq` of html strings and then use `mkString` to join them